gamemodes/terrortown/gamemode/client/cl_tbuttons.lua

```diff
@@ -56,7 +62,7 @@ end
 function TBHUD:PlayerIsFocused()
 	local ply = LocalPlayer()
 
-	return IsValid(ply) and ply:IsActive() and ply:HasTeam(TEAM_TRAITOR) and IsValid(self.focus_ent)
+	return IsValid(ply) and ply:IsActive() and ply:GetSubRoleData():CanUseTraitorButton() and IsValid(self.focus_ent)
```

Comment:
Maybe it would be better storing `CanUseTraitorButton` function and data on the player. In that way, the sidekick could also access jackal or traitor roles (regarding his mate`s team)

Reply:
Will now work for team specific buttons. If the team is set any sidekick in that team will be able to use those buttons as well.